### PR TITLE
Reorganize tests

### PR DESCRIPTION
**Pull request by joannfeng**
Could you test test_octave_action.py and let me know if it passes?

### CODE REVIEW

File: test/test_octave_action.py [hunk 1]
```diff
@@ -2,42 +2,25 @@
...
+from sos import execute_workflow
 
 
-if __name__ == '__main__':
-    unittest.main()
+@pytest.mark.skipif(not shutil.which('octave'), reason = 'Octave not installed')
+def test_octave():
+    '''Test action octave'''
+    if os.path.isfile('octave_example.txt'):
+        os.remove('octave_example.txt')
```
Comment (joannfeng) [inline test/test_octave_action.py:18]: Is this an instance where `os.remove` should be left as is?

Comment (BoPeng) [reply test/test_octave_action.py:18]: No. It should be replaced with the fixture. In your case you removed the entire `os.remove` without replacing it with the fixture.

**Comment by BoPeng** [comment]
Can you see https://travis-ci.org/github/vatlab/sos/builds/706206263 ? Your commits will be tested over there and you can see if they pass.

The entire test suite will be executed, which will take a while. If you would like to save some time, you can change [.travis.ym](https://github.com/vatlab/sos/blob/7f89b3f695edc06bcf6597bd271c69b5f355db05/.travis.yml#L62) to test a particular file, and restore it when the patch is ready to merge.

**Comment by joannfeng** [comment]
Yes I can see it, thanks!

**Comment by joannfeng** [comment]
I made the changes you described in test_convert.py and it worked.

**Comment by BoPeng** [comment]
Merged since tests pass. Thanks.

**Pull request merged**